keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -51,6 +63,7 @@
 - [Implementation History](#implementation-history)
```

Comment:
One thing we should probably add to the PRR or elsewhere is the possible need to adjust [APF](https://kubernetes.io/docs/concepts/cluster-administration/flow-control/) (from recent experience with misconfigured priority in prod 😮‍💨 )

Reply:
Why? I don't think IPPVS increases API calls, does it? Why is APF specifically relevant here?

Reply:
I guess there's the resize calls themselves, is that the case you're thinking of?

Reply:
Yeah.. I don't expect this to be a big issue. But it kubelet is lumped together with other node-infra deamonsets in the same FlowSchema, then its impact would be amplified and hard to trace. It's more of a CYA move 😉